build/teamcity/cockroach/nightlies/pebble_nightly_ycsb_race.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+#
+# This script runs the Pebble Nightly YCSB A benchmark with the race flag.
+# It is used to detect data races which may have been introduced to the latest
+# Pebble master branch.
+#
+# It is run by the Pebble Nightly YCSB A race TeamCity build
+# configuration.
+
+set -euo pipefail
```

Comment:
Likewise, you can change this line to `set -euxo pipefail` while testing for additional context.